virtuoso.sh

```diff
@@ -54,4 +58,7 @@ then
     echo `date +%Y-%m-%dT%H:%M:%S%:z` > .data_loaded
 fi
 
-exec virtuoso-t +wait +foreground
+# NOTE: put back Virtuoso's server port to 8890 before starting up
+crudini --set virtuoso.ini HTTPServer ServerPort 8890
```

Comment:
I can't find the env variable anywhere. Can you point me out where is the environment variable to configure the server's port?